lib/node_modules/@stdlib/stats/base/dists/uniform/mean/README.md

```diff
@@ -133,6 +133,96 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/uniform/mean.h"
+```
+
+#### stdlib_base_dists_uniform_mean( a, b )
+
+Returns the [mean][mean] of a [uniform][uniform-distribution] distribution with minimum value \( a \) and maximum value \( b \).
+
+```c
+double out = stdlib_base_dists_uniform_mean( 0.0, 1.0 );
+// returns 0.5
+```
+
+The function accepts the following arguments:
+
+-   **a**: `[in] double` minimum value.
+-   **b**: `[in] double` maximum value.
+
+```c
+double stdlib_base_dists_uniform_mean( const double a, const double b );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/stats/base/dists/uniform/mean.h"
+#include <stdlib.h>
+#include <stdio.h>
+
```

Comment:
You mind adding

```
static double random_uniform( const double min, const double max ) {
    double v = (double)rand() / ( (double)RAND_MAX + 1.0 );
    return min + ( v*(max-min) );
}
```

here and using it to draw the random numbers here, please? Applies to the example and benchmark C files, too.